### ISSUE TITLE
Publish to NPM

### ISSUE THREAD

**Issue by candrews** [syl22-00/pocketsphinx.js#44]
Could you please publish these libraries to NPM? Having PocketSphinx.js and recognizer.js in NPM would make them easier to use and keep up to date.

Thanks!

**Comment by syl22-00** [comment]
I an not sure it really makes sense to publish it on npm as:
* pocketsphinx.js really does make sense in the web browser. Outside a browser it is far faster to use pocketsphinx compiled natively.
* I actually did publish it on bower, but even there it was not really usable. Acoustic and Language models are often application-specific so I expect most people to compile pocketsphinx.js for their needs. However I tried to make a bunch of packages for those models, dictionaries, using bower's dependency management, to make it as easy as possible to start building applications, and keep code up to date. Any help on those would be welcome.

**Comment by derhuerst** [comment]
I'm still in favor of pocketsphinx.js being on NPM. Sometimes self-compiling is not an option.

**Comment by candrews** [comment]
NPM seems to be becoming the javascript package repository for both client and server side purposes. For example, Webpack uses NPM only by default, and jquery is published only to NPM. So if you publish to NPM, you get both client and server side - everyone wins. :)

**Comment by rwaldron** [comment]
> Sometimes self-compiling is not an option.

Yep. I wanted to preview this for Tessel 2 users, because I can't find a pocketsphinx package that will compile on openwrt.

**Comment by DanielRuf** [comment]
Bump. We also use npm for loading dependencies, like you do with composer.

**Comment by blunket** [comment]
I'd love to see this library on npm too!
